src/realm/parser/driver.cpp

```diff
@@ -1469,6 +1487,24 @@ ParserDriver::~ParserDriver()
     yylex_destroy(m_yyscanner);
 }
 
+Mixed ParserDriver::get_arg(std::string i)
+{
+    size_t arg_no = size_t(strtol(i.substr(1).c_str(), nullptr, 10));
```

Comment:
I'd recommend adding an assert that the first character is indeed a `$`.

Reply:
It is kind of promised by the lexer - but I can add it.